README.md

```diff
@@ -1 +1,108 @@
-# BCVAX-iOS
\ No newline at end of file
+# Yukon Vaccination Verifier - iOS
+
+#### The official mobile app from the Government of Yukon to verify vaccination status. This app will scan the QR code on proof of vaccination credentials to verify someone's COVID-19 vaccination status.
+
+## Table of Contents
+
+- [Project Resources](#markdown-header-project-resources)
+- [Architecture](#markdown-header-architecture)
+  - [Folder Structure](#markdown-header-folder-structure)
+  - [Libraries](#markdown-header-libraries)
+- [Configuration](#markdown-header-configuration)
+  - [Build Configurations](#markdown-header-build-configurations)
+  - [Environment Variables](#markdownpheader-environment-variables)
+  - [APIs](#markdown-header-apis)
+  - [Schemes](#markdown-header-schemes)
+  - [Running the Code](#markdown-header-running-the-code)
+- [Deployment](#markdown-header-deployment)
+  - [Versioning](#markdown-header-versioning)
+  - [Internal](#markdown-header-internal)
+  - [External](#markdown-header-external)
+- [Contributors](#markdown-header-contributors)
+
+## Project Resources
+---
+
+- [JIRA Board](https://freshworks.atlassian.net/)
+- [Designs](https://www.figma.com/)
+- [Project Overview](https://sites.google.com/freshworks.io/project-hub/projects)
+- [App Store](https://apps.apple.com/ml/app/yukon-vaccination-verifier/id1591117176)
+
+## Architecture
```

Comment:
I think we should use another name, to differ between this and the sub-bullets,  maybe "System Design" ?

Reply:
Please check now.